src/pybind/mgr/prometheus/module.py

```diff
@@ -1267,13 +1266,19 @@ def _get_pool_info(pool: Dict[str, Any]) -> Tuple[str, str]:
             if 'options' in pool:
                 compression_mode = pool['options'].get('compression_mode', 'none')
 
+            storage_type = {'rbd': 'Block', 'cephfs': 'Filesystem', 'rgw': 'Object'}
```

Comment:
maybe Enums? somewhere on the file to better manage it?

```
from enum import Enum

class StorageType(Enum):
    rbd = "Block"
    cephfs = "Filesystem"
    rgw = "Object"
```

Reply:
In that case if I have to do string conversion, I will not like to use ENUM

Reply:
I prefer enum for type safety and if the type grows in the future and we expect to add more labels to the metrics (service level granularity) but if thats something which is not being considered then dict is perfectly fine here..

And i dont think it can be `RBD`. Neither dashboard nor backend sets it like that as far as i know..

Reply:
made changes!